app/views/markets/index.html.erb

```diff
@@ -0,0 +1,30 @@
+
+
+<h1>Markets</h1>
+
+<div class="market_button">
+	<%= link_to new_market_path, class: 'btn btn-success' do %>
+	  Add a New Market
+	<% end %>
+</div>
+<br>
+<br>
+
+<table class="table table-striped" class="table table-hover">
+<tr>
+<th>Name</th>
+<th>Edit</th>
+</tr>
+<tr class="row"><% @markets.each do |market| %></tr>
+	<td><%= link_to market.name, market_path(market.id) %></td>
+
+	<td class="edit">
+    <%= link_to edit_market_path(market.id) do %>
+      <i class="fa fa-pencil" aria-hidden="true"></i>
+    <% end %>
+    </td>
+
+
+	<% end %>
+
+</table>
```

Comment:
The formatting for your table is a little off. It should be more like:
```<% @markets.each do |market| %>
<tr>
<td>market name</td>
<td>edit icon</td>
</tr>
<% end %>```
For each market, you want the form to have a new row, and within that row, you want two columns.